.circleci/config.yml

```diff
@@ -46,52 +232,41 @@ jobs:
           working_directory: ~/workbench/api
           command: ./project.rb gradle compile__bigquerytest__Java compile__integration__Java
       - run:
-          name: Run Java Unit Tests
+          name: Run Java unit tests
           working_directory: ~/workbench/api
           command: ./project.rb test
       - run:
           # Lint last; it's more important to surface test failures early.
           name: Java linting
           working_directory: ~/workbench/api
           command: ./gradlew spotlessCheck
-      - save_cache:
-          paths:
-            - ~/.gradle
-            - ~/.m2
-            - ~/workbench/api/build/exploded-api/WEB-INF/lib/
-          key: api-cache-{{ checksum "~/workbench/api/build.gradle" }}
+      - api_cache:
+          save: true
 
   api-local-test:
+    # Refers to https://circleci.com/docs/2.0/configuration-reference/#resource_class
+    resource_class: medium+
     docker:
       - image: allofustest/workbench:buildimage-0.0.16
+      # Any known reason for not using circleci/mysql:5.7-ram? better performance and mitigates I/O contention.
+      # Refers to https://circleci.com/docs/2.0/circleci-images/#service-image-variant
       - image: mysql:5.7
         environment:
           - MYSQL_ROOT_PASSWORD=ubuntu
```

Comment:
Out of curiosity, have you tried the secrets cache? I realize these aren't secret, but might be an interesting thing to explore. We probably won't get permission to use it, though.